ui/raidboss/data/05-shb/trial/diamond_weapon-ex.txt

```diff
@@ -0,0 +1,123 @@
+### Diamond Weapon Ex
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 21:........:40000010:/ window 10000 jump 0
+
+# -it "The Diamond Weapon" -ii 5FEB 5FBE 5FAA 5FEC 5FBA 5FFD 5FBB 6055 6159 5FF9 5FCD 5FFA 5FF9 612E 5FB3 5FCE 5FD2 5FFE 5FD3 5FB4 5FB6 5FB1 5FB0 5FBF 5FAE 5FA6 5FC3 5FC9 5FCA 5FC4
+
+### Phase 1
+0 "--sync--" sync /:5FA7:The Diamond Weapon/
+0 "Start" sync /Engage!/ window 1,1
+21.3 "Diamond Rain" sync /:The Diamond Weapon:5FA7:/ window 25,10
+
+# 5FA2 = Spread (Homing Laser, 5FA0)
+# 5F9B = "nothing" (Diamond Rain, 5FA7)
+# 5FA5 = Stack (Diamond Flash, 5FA1)
+36.5 "Adamant Purge" sync /:The Diamond Weapon:5FA[2345]:/ window 20,5
+46.7 "Claw Swipe" sync /:The Diamond Weapon:5F9[EF]:/
+49.9 "Homing Laser/Diamond Flash" sync /:The Diamond Weapon:5FA[01]:/
+67.1 "Photon Burst" sync /:The Diamond Weapon:5FA8:/
+83.4 "Adamant Purge" sync /:The Diamond Weapon:5F9[8B]:/
+93.6 "Claw Swipe" sync /:The Diamond Weapon:5F9[EF]:/
+104.8 "Diamond Rain" sync /:The Diamond Weapon:5FA7:/
+120.0 "Adamant Purge" sync /:The Diamond Weapon:5FA[2345]:/
+130.2 "Claw Swipe" sync /:The Diamond Weapon:5F9[EF]:/
+133.7 "Homing Laser/Diamond Flash" sync /:The Diamond Weapon:5FA[01]:/
+151.4 "Photon Burst" sync /:The Diamond Weapon:5FA8:/
+
+# Phase transition
+167.7 "Code Chi-Xi-Stigma" sync /:The Diamond Weapon:5FAD:/
+168.0 "--untargetable--"
+183.3 "Auri Cyclone" sync /:The Diamond Weapon:5FB9:/
+188.0 "Diamond Shot" sync /:Articulated Bit:5FC0:/
+
+### Phase 2
+197.8 "--targetable--"
+210.0 "Outrage" sync /:The Diamond Weapon:5FBC:/
+218.2 "Outrage" sync /:The Diamond Weapon:5FBC:/
+229.6 "Auri Arts (Z)" sync /:The Diamond Weapon:5FAF:/
+229.6 "--untargetable--"
+236.4 "Auri Arts (Jump)" sync /:The Diamond Weapon:5FB2:/
+241.0 "--targetable--"
+250.4 "Outrage" sync /:The Diamond Weapon:5FBC:/
+257.6 "Auri Doomstead" sync /:The Diamond Weapon:5FBD:/
+265.5 "--untargetable--"
+271.8 "Auri Arts (Cleave)?" sync /:The Diamond Weapon:5FB5:/
```

Comment:
```suggestion
271.8 "Auri Arts (Cleave)" sync /:The Diamond Weapon:5FB5:/
```

I'm not sure I understand why these have a question mark.

Reply:
If it's a two-orb pattern, the boss does a vertical cleave through the platform before the circle aoe. If it's one-orb, the vertical cleave doesn't happen.

Reply:
Makes sense, thanks!